confectory-core/src/main/java/net/obvj/confectory/ConfigurationContainer.java

```diff
@@ -229,7 +169,7 @@ public void clear()
      *
      * @see DataFetchStrategy
      */
-    public boolean getBoolean(String key)
+    public Boolean getBoolean(String key)
```

Comment:
Fix Javadocs (now that the wrappers are being used); also consider explaining that these methods may now return null